components/higher-order/with-api-data/test/request.js

```diff
@@ -25,15 +30,13 @@ describe( 'request', () => {
 		),
 	};
 
-	let wpApiRequest;
 	beforeEach( () => {
 		getStablePath.clear();
 		for ( const key in cache ) {
 			delete cache[ key ];
 		}
 
-		wpApiRequest = wp.apiRequest;
-		wp.apiRequest = jest.fn( () => ( {
+		apiRequest.mockReturnValue = {
```

Comment:
This part becomes a bit magic since this module is being mocked in the setup file. Have you tried to use [Manual Mocks](https://facebook.github.io/jest/docs/en/manual-mocks.html) using `__mocks__` folder? If you do so you need to explicitly call `jest.mock('./moduleName')` in your test file which is easier to follow.

Reply:
I tried `jest.mock('./moduleName', () => {})` but It didn't work. I'll take a look at the manual mocks.

Reply:
What seems to work for me is I put a `__mocks__/moduleName` within the same path as the test (so `test/__mocks__/moduleName/index.js`) and the mock is in the file then anything importing that module being tested will use the mock instead.  Seems to work well for me.

Reply:
Not that important at the moment, to be honest. Just wanted to make sure you are aware of other options. We can refactor later.